src/govuk-prototype-kit.config.json

```diff
@@ -99,6 +99,10 @@
       "macroName": "hmrcReportTechnicalIssue",
       "importFrom": "hmrc/components/report-technical-issue/macro.njk"
     },
+    {
+      "macroName": "hmrcServiceNavigationLanguageSelect",
+      "importFrom": "hmrc/components/service-navigation-language-select/macro.njk"
+    },
```

Comment:
worth a sanity check to install this into a prototype and ensure the new macro is usable

Reply:
locally built package installed fine in a new prototype, and was able to copy paste the example from description and have it display correctly

![CleanShot 2025-06-20 at 11 56 03@2x](https://github.com/user-attachments/assets/52a599cc-fa03-4a48-b049-31d1ab139ec0)